lanl/podman-quadlets/inventory/group_vars/ochami/quadlets.yaml

```diff
@@ -324,38 +322,19 @@ _haproxy_quadlets:
       - cloud-init-server.service
       - tpm-manager.service
 
-_dnsmasq_quadlets:
-  - name: dnsmasq
-    image: ghcr.io/openchami/dnsmasq:v0.1.11
+_coresmd_quadlets:
+  - name: coresmd
+    image: ghcr.io/openchami/coredhcp:v0.1.1
```

Comment:
If you want to not have an external reboot.ipxe (and thus have to have an external TFTP server), you can upgrade to v0.2.0. See my comment in the configs/templates/coredhcp/coredhcp.yaml.j2 file.

Reply:
Bumped the version to v0.2.0